### DISCUSSION TITLE
✨ Elixir: explore of using json module in OTP 27 as a default json library

### DISCUSSION THREAD

**Discussion by wingyplus** [dagger/dagger#12336] [Ideas]
### What are you trying to do?

In Erlang/OTP 27, they introduced a `json` module for encode/decode json. It would be great if we make support by default.

**Comment by wingyplus** [comment]
We may need for the Elixir 1.18 as they're planning to include JSON into stdlib

**Comment by weaversam8** [comment]
Neat! Note that for compatibility with older OTP versions you may need a polyfill: https://erlangforums.com/t/has-the-stdlib-json-module-been-backported-to-older-erlang-versions/4630
